diagnostic.md

```diff
@@ -26,7 +34,9 @@ Record your responses inside the fenced code blocks below each question.
     the syntax for loading this component inside that template?
 
     ```html
-    <!-- your response here -->
+    {{#each contact as |contact|}}
+    {{contact/my-contact contact=contact}}
+    {{/each}}
```

Comment:
you don't need to `contact/` it should look like:
`{{my-contact contact=contact}}`